hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/http/HttpServer2.java

```diff
@@ -243,7 +243,9 @@
 
     private String hostName;
     private boolean disallowFallbackToRandomSignerSecretProvider;
-    private String authFilterConfigurationPrefix = "hadoop.http.authentication.";
+    private final ArrayList<String> authFilterConfigurationPrefixes =
```

Comment:
Use List instead of ArrayList<>. It's a better practice to define fields / local variables with interface types, see: https://stackoverflow.com/questions/9852831/polymorphism-why-use-list-list-new-arraylist-instead-of-arraylist-list-n

Reply:
Thanks. Done.